staging/src/k8s.io/client-go/rest/plugin.go

```diff
@@ -18,18 +18,23 @@ package rest
 
 import (
 	"fmt"
-	"net/http"
 	"sync"
 
 	"github.com/golang/glog"
 
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"k8s.io/client-go/transport"
 )
 
 type AuthProvider interface {
-	// WrapTransport allows the plugin to create a modified RoundTripper that
-	// attaches authorization headers (or other info) to requests.
-	WrapTransport(http.RoundTripper) http.RoundTripper
```

Comment:
I think you can make this change without changing the interface since the exec provider doesn't use it. If you want to keep the diff smaller.

Reply:
Good point, reverted interface changes